### PR TITLE
[camera][web] Adding web only poster prop

### PR DESCRIPTION
**Pull request by lior-frezi**
# Why

when the camera video loads, the browser shows default "poster" for the video. on some browsers this is a blank screen, but in some web-views you get a gigantic pixelated grey play icon. anyway i thought it's a nice feature to add user control over it.

# How

looked on android only props and copied the design pattern to make a new web only prop.

# Test Plan

add a poster value to the camera and watch it displayed when the video tag loads camera input.

### CODE REVIEW

**Review by lukmccall — CHANGES_REQUESTED** [review]
Please run `yarn build` in the `expo-camera` directory and update changelog - https://github.com/expo/expo/blob/master/packages/expo-camera/CHANGELOG.md

**Comment by liorJuice** [comment]
@lukmccall did what you asked :)

File: packages/expo-camera/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- Added support for video poster to show while the camera is loading on web ([#9930](https://github.com/expo/expo/pull/9930) by [@liorJuice](https://github.com/liorJuice))
```
Comment (lukmccall) [inline packages/expo-camera/CHANGELOG.md:9]:
```suggestion
- Added support for video poster to show while the camera is loading on web. ([#9930](https://github.com/expo/expo/pull/9930) by [@liorJuice](https://github.com/liorJuice))
```

**Review by lukmccall — APPROVED** [review]
LGTM 🚀🚀🚀

**Review by bbarthec — APPROVED** [review]

**Pull request merged**